cmd/pathlist.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import 	(
+	"fmt"
+	"os"
+	"regexp"
+	"bufio"
+	"io"
+	"sort"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	regex = "data/[A-Za-z0-9_./]+\\.[a-zA-Z0-9]+"
+)
+
+var pattern regexp.Regexp
+
+func init() {
+	pathlistCmd.AddCommand(pathlistScanCmd())
+	pattern = *regexp.MustCompile(regex)
+}
+
+var pathlistCmd = &cobra.Command{
+	Use:   "pathlist",
+	Short: "Work with pathlist files",
+}
+
+func pathlistScanCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "scan",
```

Comment:
`"scan FILE"`? (and maybe update the list command, while you're at it)

Reply:
:boom: